src/docs/release-notes/R16/R16.mdx

```diff
@@ -0,0 +1,23 @@
+### Documentation Site
+
+Added search capability to doc site
+
+### New Components
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we now have components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
+-   For React Native, we now have a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
+
+### Reusable Workflows
+
+We have finished innersourcing a reusable Authentication Workflow for React Native apps, allowing teams to use a common implementation across different projects.
```

Comment:
Link to the NPM package page.

Reply:
Ah good idea, I'll add a link to the NPM package.